tg_bot.py

```diff
@@ -89,6 +95,12 @@ def button(self, update, context):
         try:
             self.twitter_api.retweet(data)
         except tweepy.TweepError as e:
+            
+            if self.level == 50:
```

Comment:
what does these values denote? Perhaps add a comment to document what this means?

Reply:
Instead of setting the values for each args we can set the value as args as it is

Reply:
There are few inbuilt macros which signifies these level, which will be more human-understandable. Also ill add a link which the developers can refer if needed.